src/clj_pgp/core.clj

```diff
@@ -285,7 +300,7 @@
   [data]
   (with-open [stream (PGPUtil/getDecoderStream
                        (bytes/to-input-stream data))]
-    (doall (read-objects stream))))
+    (into '() (read-objects stream))))
```

Comment:
You could use [`sequence`](https://clojuredocs.org/clojure.core/sequence) to present the same seq interface, but it's not really important that this be a list. Since we're building it eagerly, a vector is a better choice here.